CONTRIBUTING.md

```diff
@@ -55,6 +55,14 @@ git checkout -b cool-new-feature
 
 4\. When you are ready, submit a pull request into the composer repository! If merged, we'll reach out to send you some free swag :)
 
+## Pull request guidelines
+
+We have some rough guidelines that will make your PR easier to review and more likely to get smoothly merged.
+* Self-contained title and description. Please include a descriptive title and description. These should allow someone to understand the contents of the PR without looking at the code.
+* If the PR affects output that is displayed to a user of Composer (e.g. console logging or experiment tracker reporting), please include screenshots showing what the new output looks like. UX is important!
+* Include tests. If you are fixing a bug, please add a test that would've caught the bug. If you are adding a new feature, please add unit tests that test the various components of the feature, and also a test that tests the full functionality of the feature.
```

Comment:
Should we include something about manual testing as well?  For example, we don't run the full test suite in PR because we want PR CI tests to run quickly.  Ideally it would be good if developers ran other tests that aren't run by the PR CI (e.g., things we run in the `daily`).  I don't think this is a requirement as there's no easy way to enforce it other than just running the tests, but it is good practice.